app/views/games/show.html.erb

```diff
@@ -28,95 +63,96 @@
+          <div class="black left"><%= render_piece(7,4) %></div>
+        </div>
 
-      <div class="row">
-        <div class="black left"><%= render_piece(0,3) %></div>
-        <div class="white left"><%= render_piece(1,3) %></div>
-        <div class="black left"><%= render_piece(2,3) %></div>
-        <div class="white left"><%= render_piece(3,3) %></div>
-        <div class="black left"><%= render_piece(4,3) %></div>
-        <div class="white left"><%= render_piece(5,3) %></div>
-        <div class="black left"><%= render_piece(6,3) %></div>
-        <div class="white left"><%= render_piece(7,3) %></div>
-      </div>
+        <div class="row">
+          <div class="black left"><%= render_piece(0,3) %></div>
+          <div class="white left"><%= render_piece(1,3) %></div>
+          <div class="black left"><%= render_piece(2,3) %></div>
+          <div class="white left"><%= render_piece(3,3) %></div>
+          <div class="black left"><%= render_piece(4,3) %></div>
+          <div class="white left"><%= render_piece(5,3) %></div>
+          <div class="black left"><%= render_piece(6,3) %></div>
+          <div class="white left"><%= render_piece(7,3) %></div>
+        </div>
 
-      <div class="row">
-        <div class="white left"><%= render_piece(0,2) %></div>
-        <div class="black left"><%= render_piece(1,2) %></div>
-        <div class="white left"><%= render_piece(2,2) %></div>
-        <div class="black left"><%= render_piece(3,2) %></div>
-        <div class="white left"><%= render_piece(4,2) %></div>
-        <div class="black left"><%= render_piece(5,2) %></div>
-        <div class="white left"><%= render_piece(6,2) %></div>
-        <div class="black left"><%= render_piece(7,2) %></div>
-      </div>
-      <div class="row">
-        <div class="black left"><%= render_piece(0,1) %></div>
-        <div class="white left"><%= render_piece(1,1) %></div>
-        <div class="black left"><%= render_piece(2,1) %></div>
-        <div class="white left"><%= render_piece(3,1) %></div>
-        <div class="black left"><%= render_piece(4,1) %></div>
-        <div class="white left"><%= render_piece(5,1) %></div>
-        <div class="black left"><%= render_piece(6,1) %></div>
-        <div class="white left"><%= render_piece(7,1) %></div>
-      </div>
+        <div class="row">
+          <div class="white left"><%= render_piece(0,2) %></div>
+          <div class="black left"><%= render_piece(1,2) %></div>
+          <div class="white left"><%= render_piece(2,2) %></div>
+          <div class="black left"><%= render_piece(3,2) %></div>
+          <div class="white left"><%= render_piece(4,2) %></div>
+          <div class="black left"><%= render_piece(5,2) %></div>
+          <div class="white left"><%= render_piece(6,2) %></div>
+          <div class="black left"><%= render_piece(7,2) %></div>
+        </div>
+        <div class="row">
+          <div class="black left"><%= render_piece(0,1) %></div>
```

Comment:
For each square, you'll want to know the x/y coordinates. So you can add them to the div as `data` attributes. These will be the squares that are "droppable". When you drop you can use the data attributes to know where the user is trying to move the piece to